src/styles/arduino-dark.css

```diff
@@ -0,0 +1,86 @@
+/*
+
+  Arduino® Dark Theme - BelKed (https://github.com/BelKed)
+
+*/
+
+.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #282c34;
+}
+
+.hljs-ln-n {
+  color: #555555;
+}
+
+.hljs-ln-code {
+  color: #ffffff;
+}
+
+.hljs-params {
+  color: #aaaaaa;
+}
+
+.hljs,
+.hljs-subst {
+  color: #434f54;
+}
+
+.hljs-keyword,
+.hljs-attribute,
+.hljs-selector-tag,
+.hljs-doctag,
+.hljs-name,
+.hljs-built_in,
+.hljs-literal,
+.hljs-bullet,
+.hljs-code,
+.hljs-addition,
+.hljs-regexp,
+.hljs-symbol,
+.hljs-variable,
+.hljs-template-variable,
+.hljs-link,
+.hljs-selector-attr,
+.hljs-selector-pseudo,
+.hljs-type,
+.hljs-string,
+.hljs-selector-id,
+.hljs-selector-class,
+.hljs-quote,
+.hljs-template-tag,
+.hljs-deletion {
+  color: #61afef;
+}
```

Comment:
This is a bit unexpected... can't we follow the same patterns used over in the light font to make sure that this also works equally well highlighting HTML, CSS, etc, not just source code?

Reply:
OK, I edited it in https://github.com/highlightjs/highlight.js/pull/2977/commits/3452ccfa61a2109c4043a7338452d3b412fba3ab.